tests/test_solvers/test_harbor_terminus_patches.py

```
@@ -445,19 +450,113 @@ def test_terminus2_triggers_all_patches(self, tmp_path, monkeypatch):
+        "You passed 262145 input tokens and requested 0 output tokens. "
+        "However, the model's context length is only 262144 tokens, "
+        "resulting in a maximum input length of 262144 tokens. Please "
+        "reduce the length of the input prompt. "
+        "(parameter=input_tokens, value=262145)"
+    )
+
+    @staticmethod
+    def _fake_bad_request(message: str) -> SimpleNamespace:
+        return SimpleNamespace(body="", message=message, error="")
+
+    def _install(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_install`

(ANN202)